analyses/immune-deconv/02-visualize_quantiseq.Rmd

```diff
@@ -0,0 +1,225 @@
+  scale_color_identity() +
+  labs(
+    x = "Molecular subtype of tumor sample", 
+    y = "Estimated fraction in sample"
+  ) +
+  ggpubr::theme_pubr() +
+  theme(axis.text.x = element_text(angle = 45, hjust = 1, size = 6),
+        axis.text.y = element_text(size = rel(0.7)),
+        strip.text = element_text(size = rel(0.7)))
+
+subtypes_celltypes_plot
+
+ggsave(subtypes_celltypes_plot_file, subtypes_celltypes_plot, width = 8, height = 6)
+```
+
+
+### Visualize PDL1 (CD274) expression
+
+In [this](https://www.tandfonline.com/doi/full/10.1080/2162402X.2018.1462430) reference, high levels of both CD8+ and PDL1 can mean good candidate for immunotherapy, and high levels of PDL1 with low levels of CD8+ in has been associated with poor prognosis.
+
+In our data, CD8+ fractions are very low, at most 1.5% in an HGG H3 WT sample. Distributions of CD8+ fractions also show extremely low variance, _except_ for MB subtypes (relatively speaking). Therefore, we examine here the relationship between expression and fraction for MB specifically.
+
+
+
+First, we explore the relationship between variables with a scatterplot. Because there is very little variation in CD8+, this is not terribly informative.
+
+```{r pdl1_scatterplot, fig.width = 10, fig.height = 6}
+# Merge and subset data to get log2 expression of PDL1 and CD8 fractions
+pdl1_cd8 <- expression_pdl1 %>%
+  inner_join(quantiseq_subset) %>%
+  select(log2_expression, cell_type, library, score, molecular_subtype, cancer_group_display, cancer_group_hex) %>%
+  filter(cell_type == "T cell CD8+", 
+         cancer_group_display != "Other") 
+
+
+# scatterplot of CD8 fractions and PDL1 expression.
+ggplot(pdl1_cd8) + 
+  aes(x = score, y = log2_expression, color = library) + 
+  geom_jitter() + 
+  facet_wrap(~molecular_subtype, nrow = 5) +
+  labs(x = "CD8+ fraction in sample", y = "log2 PDL1 expression") +
+  ggpubr::theme_pubr() + 
+  theme(axis.text.x = element_text(hjust = 1, 
+                                   size = rel(0.8), 
+                                   angle = 45)) +
+  cowplot::panel_border() 
+```
+
+
+Instead, let's explore the PDL1 distributions specifically and export this figure:
+
+```{r pdl1_distributions, fig.width = 8, fig.height = 8}
+
+# distributions of PDL1 expression
+pdl1_distributions_plot <- ggplot(pdl1_cd8) + 
+  aes(x = fct_reorder(molecular_subtype,log2_expression, .desc=T),
+      y = log2_expression) +
+  # remove outliers
+  geom_boxplot(outlier.shape = NA, color = "grey40", size = 0.2) + 
+  geom_jitter(width = 0.15, size = 0.75, alpha = 0.8, aes(color = cancer_group_hex)) + 
+  facet_wrap(~cancer_group_display, nrow = 2, scales = "free") +
+  scale_color_identity() +
+  labs(x = "Molecular subtype of sample",
+       y = "Log2 expression of PDL1") +
+  ggpubr::theme_pubr() + 
+  theme(axis.text.x = element_text(hjust = 1, 
+                                   size = rel(0.7), 
+                                   angle = 45)) 
```

Comment:
```suggestion
                                   angle = 45)) 
```
 Can you either make this a higher angle or add a margin to the left side since LGG, BRAF V600E, CDKN2A/B is getting cut off?